MapboxNavigation/NavigationMapView.swift

```diff
@@ -56,6 +56,16 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
      */
     public weak var courseTrackingDelegate: NavigationMapViewCourseTrackingDelegate?
     
+    /**
+     The possible routes that can be represented on the map view.
+     */
+    public var routes: [Route]?
+    
+    /**
+     The user location the map view should follow.
+     */
+    @objc public var userLocationForCourseTracking: CLLocation?
```

Comment:
Same deal: we should avoid making this property public, or at least avoid making it discoverable.